compiler/rustc_target/src/spec/base/aix.rs

```diff
@@ -22,10 +23,12 @@ pub fn opts() -> TargetOptions {
         is_like_aix: true,
         default_dwarf_version: 3,
         function_sections: true,
-        pre_link_objects: crt_objects::new(&[
```

Comment:
The lazy part can be moved to `crt_objects::new` if CRT objects are always lazy.